include/lbann/models/model.hpp

```diff
@@ -428,6 +428,11 @@ class model {
    */
   std::string m_name;
 
+  /** @brief Is the model setup
+   *  @details Flag to indicate if the setup function has been called
+   */
+  bool m_model_setup;
```

Comment:
Put this somewhere else. Order members by increasing or decreasing size for optimal alignment potential.